vuepress/docs/next/docs/releases/README.md

```diff
@@ -1,10 +1,197 @@
+* Moved from a HELM based deployment process to a pure CustomResource based deployment and installation process.
+* Installation parameters considered by the operator all moved to the entando-operator-config ConfigMap.
+* Better support for cloud DBMS.
+* Updated libraries (Jetty, Keycloak, Spring) and the entire compatibility matrix.
+* Dropped support for suffix-mode installation. Only single-hostname is supported now.
+* Temporarily dropped support for cluster-level operator installation.
+* Removed EntandoClusterInfrastructure and EntandoCompositeApp CR.
+* k8s-service is now part of the base Entando infrastructure and therefore installed alongside the operator.
+* ComponentManager now communicates with k8s-service using service-to-service authentication, and strictly within the cluster network.
+* Entando Operator deployment is now simply called entando-operator.
+* k8s-service deployment dropped the application qualifier and is now simply called entando-k8s-service.
+* Fixed an intermittent issue with the operator that prevented installing new bundles until the operator was restarted.
+* Updated all the Entando base images for reliability and security.
+* Replaced the centos images to rocky images and rebased the derived images.
+* Updated the auto provisioned PostgreSQL image to version 14.
+* MySQL and PostgreSQL are now served by customized and updated entando-mysql and entando-postgres images.
+
+### Entando Bundler
 
-Entando implements the SQL versioning tool Liquibase to track, manage and deploy updates to databases. Refer to [Liquibase Migration](../reference/databases.md#liquibase-migration) to learn how the App Engine automates database schema management and modifications.
+* It’s now possible to extract protected assets.
+* Fixed several bugs in the extraction of pages and widgets.
+* Added support for categories.
+* Error handling has been improved.
+* It's now possible to generate the so-called system-level bundles (see ECR).
+* Extract/download optimized to only touch the relevant resources.
+* Libraries updated for reliability and security.
+
+### EOL Kubernetes Versions
+
+Entando7 drops official support for for OS3.11 and Kubernetes versions prior to 1.20.
+
+### Google Kubernetes Engine (GKE)
+
+* Postponed support to 7.0.1.
+
+### Azure Kubernetes Service (AKS)
+
+* Postponed support to 7.0.1.
+
+## Breaking Changes
+
+* The internal PostgreSQL version has been updated to version 14. Bundles that rely on the auto provisioning of PostgreSQL may (or may not) be affected.
+
+## Known Issues
+
+### ECG
+
+* The default "details" widget generated for an entity doesn't work out of the box.
+* Local Keycloak loses state across restarts when using the `--no-recreate` option.
+
+### ECR
+
+* An in-place upgrade from version 6.3.X to version 7 of the ECR is not possible, but a simple migration script will be provided. In-place upgrades from version 7 to future versions will be possible.
```

Comment:
migration script -> upgrade script.  We should use the work upgrade instead of migration since migration implies a platform migration and generally means a lot more work 

@w-caffiero-entando is there a ticket for the upgrade script?  Was this included in the upgrade scripts @zonia3000 worked up?

Reply:
@nshaw No, my scripts upgrade the de-app database. Maybe @firegloves is working on that?

Reply:
Yes, I worked on that script, but we don't have a ticket at the moment. Currently I'm the only owner of the script, please let me know where I can upload it

Reply:
@firegloves I can't help with the ticket but I like the pattern we settled on for the de-app. Basically an upgrade folder with a very clear pattern: https://github.com/entando/entando-de-app/tree/develop/upgrade. Could we do something similar for the ECR on the appropriate repo?  CC: @w-caffiero-entando

Reply:
@nshaw agree, but in the CM repository.
@firegloves get in touch with NP for the ticket

Reply:
@w-caffiero-entando @nshaw added [here](https://github.com/entando-k8s/entando-component-manager/tree/develop/upgrade/6.3.2-to-7.0.0/), please take a look

Reply:
@firegloves looks good, I added a note to the ticket for the upgrade - https://entando.myjetbrains.com/youtrack/issue/ENDOC-444. @Lyd1aCla1r3 let's discuss where we can collect the upgrade information.